examples/api/lib/painting/axis_direction/axis_direction.0.dart

```diff
@@ -0,0 +1,208 @@
+        axis = Axis.horizontal;
+      case AxisDirection.right:
+        reverse = false;
+        axis = Axis.horizontal;
+      case null:
+    }
+    setState((){
+      // Respond to change in axis direction.
+    });
+  }
+
+  Widget _getLeading() {
+    return Container(
+      color: Colors.blue[100],
+      child: Padding(
+        padding: const EdgeInsets.all(8.0),
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: <Widget>[
+            Text(axis.toString()),
+            spacer,
+            Text(_getAxisDirection().toString()),
+            spacer,
+            const Text('GrowthDirection.forward'),
+            spacer,
+            _getArrows(),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _getRadioRow() {
+    return DefaultTextStyle(
+      style: const TextStyle(fontWeight: FontWeight.bold, color: Colors.white),
+      child: RadioTheme(
+        data: RadioThemeData(
+          fillColor: MaterialStateProperty.all<Color>(Colors.white),
+        ),
+        child: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: <Widget>[
+              Radio<AxisDirection>(
+                value: AxisDirection.up,
+                groupValue: _getAxisDirection(),
+                onChanged: _onAxisDirectionChanged,
+              ),
+              const Text('up'),
+              spacer,
+              Radio<AxisDirection>(
+                value: AxisDirection.down,
+                groupValue: _getAxisDirection(),
+                onChanged: _onAxisDirectionChanged,
+              ),
+              const Text('down'),
+              spacer,
+              Radio<AxisDirection>(
+                value: AxisDirection.left,
+                groupValue: _getAxisDirection(),
+                onChanged: _onAxisDirectionChanged,
+              ),
+              const Text('left'),
+              spacer,
+              Radio<AxisDirection>(
+                value: AxisDirection.right,
+                groupValue: _getAxisDirection(),
+                onChanged: _onAxisDirectionChanged,
+              ),
+              const Text('right'),
+              spacer,
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('AxisDirection'),
+        bottom: PreferredSize(
+          preferredSize: const Size.fromHeight(50),
+          child: Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: _getRadioRow(),
+          ),
+        ),
+      ),
+      body: CustomScrollView(
+        reverse: reverse,
+        scrollDirection: axis,
+        slivers: <Widget>[
+          SliverList.builder(
+            itemCount: 27,
+            itemBuilder: (BuildContext context, int index) {
+              late Widget child;
```

Comment:
```suggestion
              final Widget child;
```

(I believe dart's flow analysis is smart enough to recognize that all code paths will assign this before it is used)

Reply:
same for the other examples